source/sass/blocks/navigation.scss

```
@@ -1,41 +1,41 @@
 .navigation {
   position: relative;
   display: grid;
-  grid-template-columns: [menu-start logo-start] 1fr [logo-end] 1fr [menu-end];
+  grid-template-columns: 1fr 1fr;
   align-items: center;
 
   min-height: 79px;
 
   @media (min-width: $tablet-width) {
-    grid-template-rows: [menu-start logo-start] 89px [logo-end] 1fr [menu-end];
+    grid-template-rows: 89px  1fr;
+    grid-template-areas:
+      "logo  user"
+      "site  del";
   }
 
   @media (min-width: $desktop-width) {
-    grid-template-columns: [menu-start] 1fr [logo-start] 1fr [logo-end] 1fr [menu-end];
+    grid-template-columns: repeat(3, 1fr);
+    grid-template-areas:
+      "site  logo  user"
+      "sale  .     del";
```

`del`? А что там было про удаление...

---

Можно использовать свободное место под меню, чтобы оно переполнялось сложнее
```suggestion
      "sale sale del";
```